display.js

```diff
@@ -49,21 +49,22 @@ function createExtracts(results) {
 function activatePage() {
     var spinner = createSpinner();
     RESULTSBOX.appendChild(spinner);
-    return getLocations().then(
+    getLocations().then(
```

Comment:
the return has to be here so that it returns a promise
```suggestion
    return getLocations().then(
```